sentry-android-core/src/test/java/io/sentry/android/core/AppStartStateTest.kt

```diff
@@ -0,0 +1,44 @@
+package io.sentry.android.core
+
+import java.util.Date
+import kotlin.test.BeforeTest
+import kotlin.test.Test
+import kotlin.test.assertNull
+import kotlin.test.assertSame
+
+class AppStartStateTest {
+
+    @BeforeTest
+    fun `reset instance`() {
+        AppStartState.getInstance().resetInstance()
+    }
+
+    @Test
+    fun `appStartInterval returns null if end time is not set`() {
+        val sut = AppStartState.getInstance()
+
+        sut.setAppStartTime(0, Date())
```

Comment:
`l`: Using always the same date makes the test deterministic.
```suggestion
        sut.setAppStartTime(0, Date(0))
```